src/pages/theming/dark-mode.md

```diff
@@ -0,0 +1,121 @@
+---
+previousText: 'Advanced'
+previousUrl: '/docs/theming/advanced'
+contributors:
```

Comment:
Shouldn't there be a next page here still? Notice advanced goes on to publishing.

I think it should actually switch order though so dark mode is before advanced.

Reply:
My though is that since dark mode is kind of an advanced feature, it should come after. We can add next buttons and send it to publishing from dark mode?